src/v2/styles/AttachmentList/AttachmentList-layout.scss

```diff
@@ -346,7 +352,7 @@
     grid-template-columns: 36px auto 55px;
     align-items: center;
     justify-content: center;
-    gap: calc(var(--str-chat__spacing-px) * 14);
+    gap: calc(var(--str-chat__spacing-px) * 9);
```

Comment:
Made this value to smaller to make sure that 40 bar fits a desktop screen with our default message width of `300px`

Reply:
![Screenshot 2024-03-21 at 14 34 00](https://github.com/GetStream/stream-chat-css/assets/6690098/066377f1-a23d-4c22-a998-01518adab2fc)